src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -182,23 +209,18 @@ export default {
 		// Get Lend Filter Exp version
 		this.getLendFilterExpVersion();
 
-		// EXP to Hide filters not available in FLSS (VUE-970)
-		const lendFilterFlssExp = this.apollo.readFragment({
-			id: 'Experiment:lend_filter_flss_mock',
-			fragment: experimentVersionFragment,
-		}) || {};
-
-		if (lendFilterFlssExp.version && lendFilterFlssExp.version !== 'unassigned') {
-			if (lendFilterFlssExp.version === 'b') {
-				this.hideNonFlssFilters = true;
-				this.initiallyExpandedFilters = true;
+		if (isFLSSEligible(this.$route)) {
+			const { enabled } = getExperimentSettingCached(this.apollo, lendFilterRedirectEXP);
+			if (enabled) {
+				// this method will get the version from the apollo cache
+				trackExperimentVersion(
```

Comment:
Is there a reason we track this here instead of on the FLSS filter page?

Reply:
This event will fire for visitors in the control group of the experiment with `a` as the "version" whereas the event on our new page will fire with `b` as the version. This page will redirect before it's loaded in the browser if the are in the experiment group and the event fires in the client.

Reply:
Ah, so we're tracking eligibility regardless.